static/style.css

```diff
@@ -0,0 +1,130 @@
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.navbar {
+  width: 100%;
+  overflow: auto;
+  justify-content: space-between;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  padding: 10px 24px;
+  justify-content: space-between;
+  background-color: white;
+  position: fixed;
+  width: 100%;
+}
+
+main {
+  overflow-y: auto;
+}
+
+.button {
+  position: absolute;
+  top: 100%;
+}
+
+.bio {
+  height: 718px;
+  width: 100%;
+  background-color: #fff;
+  background-size: contain;
+  background-position: center;
+  padding: 100px 24px;
+  background-origin: border-box;
+  border-bottom-left-radius: 30%;
+}
+
+.bio-header {
+  font-family: Playfair Display, sans-serif;
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 50px;
+  letter-spacing: 0;
+  text-align: left;
+  padding: 0 4px;
+}
+
+.bio-info {
+  font-family: Poppins, sans-serif;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  text-align: left;
+  padding: 10px 5px;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  object-fit: contain;
+  background-size: cover;
+  height: auto;
+  padding: 60px 10px;
+}
+
+@media screen and (min-width: 375px) {
+  .menu-icon {
+    display: block;
+  }
+
+  .link-text {
+    display: flex;
+    background-color: #1a2236;
+    border: none;
+    color: white;
+    padding: 15px 100px;
+    text-align: center;
+    text-decoration: none;
+    font-size: 16px;
+    margin: 4px 15px;
+    cursor: pointer;
+    align-items: center;
+    justify-content: center;
```

Comment:
Kindly adjust button to match the design. It's aligned to the left with the headline text and the "Start Collaboration" text is on a single line.

Please, also adjust the margin of the header section from the container.

Your design:
![image](https://user-images.githubusercontent.com/20554583/235740008-3530ecc7-b269-4697-8893-7669a43a25be.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/235740418-6e2c91ad-fac0-4da7-a86a-f3179afbe805.png)